### ISSUE TITLE
Issues using ob-julia

### ISSUE THREAD

**Issue by fcdimitr** [gcv/julia-snail#119]
Thank you very much for this great package!

I am using `doom-emacs` with minor customizations. I have installed `julia-snail` and am trying to use `ob-julia` in `org`. 

The issue is that anytime I try to evaluate a code block, the `org-babel-execute:julia` function becomes redefined by the `ob-julia` packaged with `org`. I have yet to find a way to force to use the version that comes with this package. I am currently unable to run the `ob-julia` version that ships with this package.

Do you have any ideas or workarounds on how to resolve this issue?

This may not be an issue of `julia-snail`, but I am not sure where to post it. 

Thanks!

**Comment by gcv** [comment]
It seems like it should be possible to patch this, either directly in Snail, or by coordinating with whoever maintains the code which overrides the Snail Babel integration.

@MasonProtter, what do you think?

**Comment by aosidnf** [comment]
maybe these patches for the doom repo are helpful for you?
http://0x0.st/H8C6.patch
http://0x0.st/H8C0.patch

**Comment by RomeoV** [comment]
I haven't been able to get it to work, even with the patches..

**Comment by RomeoV** [comment]
However, actually hlissner (the doom emacs creator) posted a simple fix [here](https://github.com/doomemacs/doomemacs/issues/6327#issuecomment-1107834455):

```elisp
;;; add to $DOOMDIR/config.el
(add-to-list '+org-babel-mode-alist '(julia . julia-snail))
```

**Comment by Emiller88** [comment]
He's the best!

**Comment by gcv** [comment]
I don't use Doom Emacs, so I don't fully follow what happened. Does this mean this ticket can be closed as solved?

**Comment by RomeoV** [comment]
I'm considering making a PR to the doomemacs repo to put it into the `julia-snail` config, but I'm actually surprised that you don't need it in Vanilla emacs. Would it make  sense to include this line in this repo here directly?

**Comment by gcv** [comment]
Not as far as I can tell. Using Emacs 29.1 and its included Org 9.6.6, I don't see any sign of a `org-babel-execute:julia` function. I think `+` prefixes are a Doom thing, so `+org-babel-mode-alist` should not be added to a non-Doom codebase. I also don't see anything resembling `org-babel-mode-alist` in my Emacs environment. The `ob-julia` Snail extension works fine for me without any of these changes. 🤷

**Comment by RomeoV** [comment]
Then you can close this issue imo. Thanks!
